runtime/src/process/metered.rs

```diff
@@ -113,4 +112,16 @@ mod tests {
         let new_virt = metrics.virtual_memory.get();
         assert!(new_virt > virt, "Virtual memory should be > {virt}");
     }
+
+    #[test]
+    fn test_process_metrics_init() {
```

Comment:
I'd prefer we either just check both are > 0 or loop indefinitely until passing.

We try to avoid these timeout/retry style tests as they are inherently flaky.

(In any case, good find!)